proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtils.java

```diff
@@ -307,12 +361,16 @@ private static String fromAnyValue(AnyValue anyValue) {
       return values.stream().map(OtlpProtobufUtils::fromAnyValue)
           .collect(Collectors.joining(", ", "[", "]"));
     } else if (anyValue.hasKvlistValue()) {
-      if (OTLP_DATA_LOGGER.isLoggable(Level.FINEST)) {
-        OTLP_DATA_LOGGER.severe("Encountered KvlistValue but cannot convert to String");
-      }
+      log(Level.SEVERE, "Encountered KvlistValue but cannot convert to String");
     } else if (anyValue.hasBytesValue()) {
       return Base64.getEncoder().encodeToString(anyValue.getBytesValue().toByteArray());
     }
     return "<Unknown OpenTelemetry attribute value type " + anyValue.getValueCase() + ">";
   }
+
+  private static void log(Level level, String message) {
```

Comment:
why?

Reply:
I found it noisy having a bunch of conditionals in the code containing `if (OTLP_DATA_LOGGER.isLoggable(Level.FINEST)`, so I thought it cleaner to extract a method to do the logging.

Reply:
Yes, is noisy, but it have its propourse which is to prevent the string message to be build (concatenated).
The `OTLP_DATA_LOGGER.log(level, message);` first this that call do is checking if the log need to be send, which you already did it.

That's why this make sense (don't look to the finest/info thing ;-) ):
```
    if (ZIPKIN_DATA_LOGGER.isLoggable(Level.FINEST)) {
      ZIPKIN_DATA_LOGGER.info("Converted Wavefront span: " + wavefrontSpan.toString());
    }
```
If the level is not finest you don't call `toString` which could be expensive. 
But with your method, your call `toString` allways.